### PR TITLE
Sign up I - i18n

### PR DESCRIPTION
**Pull request by Julianhm9612**
## Summary

- Update i18n dependencies
- Update  i18n and yup locales configuration
- Add language component to change the languages
- Add error message component for inputs
- Update input component with new error message component
- Update sign up component whit i18n

## Screenshots

i18n

## Trello Card

https://trello.com/c/1THrsJE0/21-internationalization

### CODE REVIEW

File: src/components/Language/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.language {
+  bottom: 16px;
+  left: 16px;
+  position: absolute;
+}
+
+.selectLang {
+  border: 1px solid #00ADEE;
```
Comment (ebedoyaalzate) [inline src/components/Language/styles.module.scss]: add color to variables

File: src/components/Language/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  border-radius: 5px;
+  color: black;
```
Comment (ebedoyaalzate) [inline src/components/Language/styles.module.scss]: same here, with hexadecimal values

**Review by ebedoyaalzate — APPROVED** [review]

File: src/components/Language/styles.module.scss (hunk 2 above)
Comment (Julianhm9612) [reply src/components/Language/styles.module.scss]: Fixed too

File: src/components/Language/styles.module.scss (hunk 1 above)
Comment (Julianhm9612) [reply src/components/Language/styles.module.scss]: Color variables imported :rage2:

**Review by ebedoyaalzate — APPROVED** [review]
👌

**Pull request merged**